### PR TITLE
Update various files

### PR DESCRIPTION
**Pull request by bertrand-lorentz**
Various changes that will go into SDK 1.6.0.

See the individual commits for details.

### CODE REVIEW

**Review by YvesJo — DISMISSED** [review]
Looks ok to me.
Just the controls should be updated to also consider Groups and not just Nodes

**Review by pdonohoe — APPROVED** [review]
Corrected rule|text|ND-LotTenderingProcess-14 and group|name|ND-LotDistribution

Is there a reason that [$isChangeNotice = true()] is not used in this location?
validation-stage-6a.sch: <rule context="/*/cac:ProcurementProject/cac:MainCommodityClassification[/*/ext:UBLExtensions/ext:UBLExtension/ext:ExtensionContent/efext:EformsExtension/efac:Changes/efbc:ChangedNoticeIdentifier]">

File: translations/rule_en.xml [hunk 1]
```diff
@@ -38206,7 +38206,7 @@
 <entry key="rule|text|ND-LotTenderingProcess-11">Information under "Contracting system" is missing (element cac:TenderingProcess is mandatory under /*/cac:ProcurementProjectLot[cbc:ID/@schemeName='Lot'])</entry>
 <entry key="rule|text|ND-LotTenderingProcess-12">Information under "Contracting system" is missing (element cac:TenderingProcess is mandatory under /*/cac:ProcurementProjectLot[cbc:ID/@schemeName='Lot'])</entry>
 <entry key="rule|text|ND-LotTenderingProcess-13">Information under "Contracting system" is missing (element cac:TenderingProcess is mandatory under /*/cac:ProcurementProjectLot[cbc:ID/@schemeName='Lot'])</entry>
-<entry key="rule|text|ND-LotTenderingProcess-14">Information under "Purpose" is missing (element cac:TenderingProcess is mandatory under /*/cac:ProcurementProjectLot[cbc:ID/@schemeName='Lot'])</entry>
+<entry key="rule|text|ND-LotTenderingProcess-14">Information under "Procedure" is missing (element cac:TenderingProcess is mandatory under /*/cac:ProcurementProjectLot[cbc:ID/@schemeName='Lot'])</entry>
```
Comment (YvesJo) [inline translations/rule_en.xml:38209]: Not sure this is a  recommended change as Procedure refers generally to the whole procedure while here the rule is about a specific lot (/*/cac:ProcurementProjectLot[cbc:ID/@schemeName='Lot'])

File: translations/group_en.xml [hunk 2]
```diff
@@ -262,7 +262,7 @@
 <entry key="group|name|ND-LotAwardCriterionParameter">Award criterion</entry>
 <entry key="group|name|ND-LotAwardCriterionParameters">Award criteria</entry>
 <entry key="group|name|ND-LotAwardCriterionTypeUnpublish">Not Immediately Published (LotAwardCriterionType)</entry>
-<entry key="group|name|ND-LotDistribution">Lots Distribution Information (Max number of Lots a same tenderer may win, Max number of lots a tenderer may tender for, Group of lots and their composition)</entry>
+<entry key="group|name|ND-LotDistribution">Lots Distribution Information (Max number of Lots the same tenderer may win, Max number of lots a tenderer may tender for, Group of lots and their composition)</entry>
```
Comment (YvesJo) [inline translations/group_en.xml:265]: As this is defined even before any tenderer exists (at competition) I would prevent the use of the definite article. May be go for "a tenderer" instead?

File: translations/rule_en.xml (hunk 1 above)
Comment (pdonohoe) [reply translations/rule_en.xml:38209]: OK. But "Purpose" is wrong. Perhaps we could use "Process" instead?

File: translations/group_en.xml (hunk 2 above)
Comment (pdonohoe) [reply translations/group_en.xml:265]: Yes - "a tenderer" is fine. It is one of the strange / stupid parts of English that "a same" is never correct, it should always be "the same", even when there is no specific subject.

File: translations/rule_en.xml (hunk 1 above)
Comment (YvesJo) [reply translations/rule_en.xml:38209]: Indeed, Purpose is definitely wrong as it entails the description of the lot and not the process/procedure associated to it. Process could be better

File: translations/group_en.xml (hunk 2 above)
Comment (YvesJo) [reply translations/group_en.xml:265]: Thanks for the explanation.

Comment (pdonohoe) [reply translations/group_en.xml]: OK I have updated it to "a tenderer"

**Review by YvesJo — DISMISSED** [review]

**Review by pdonohoe — APPROVED** [review]

**Comment by bertrand-lorentz** [comment]
> Is there a reason that [$isChangeNotice = true()] is not used in this location?
> validation-stage-6a.sch:

This file is generated from the rules defined in the DB, based on a template. I didn't want this to depend on a variable, and it's OK if XPaths are a bit long here, as they don't need to be edited manually.

File: fields/fields.json [hunk 3]
```diff
@@ -22338,6 +22344,7 @@
     "xpathRelative" : "cbc:ID",
     "type" : "id",
     "idScheme" : "TPA",
+    "schemeName" : "Tendering-party",
```
Comment (YvesJo) [inline fields/fields.json:22356]: Should be "**t**endering-party" instead

**Review by YvesJo — COMMENTED** [review]
Just a schemeName value to be fixed

File: fields/fields.json (hunk 3 above)
Comment (bertrand-lorentz) [reply fields/fields.json]: Good catch, I've corrected this in the DB, and re-generated fields.json (see added commit).

**Review by YvesJo — APPROVED** [review]

**Comment by bertrand-lorentz** [comment]
Merging without waiting for re-approval, as it was approved before and I just updated examples.
There will be other, new PRs before 1.6.0.

**Pull request merged**